input/pagecontent/terminology.md

```diff
@@ -8,72 +8,68 @@ This implementation guide does not define any unique value sets. The value sets
-- [Condition/Diagnosis Severity](https://hl7.org/fhir/R4/valueset-condition-severity.html)
-- [ConditionCategoryCodes](https://hl7.org/fhir/R4/valueset-condition-category.html)
-- [ConditionClinicalStatusCodes](https://hl7.org/fhir/R4/valueset-condition-clinical.html)
-- [ConditionVerificationStatus](https://hl7.org/fhir/R4/valueset-condition-ver-status.html)
-- [DataAbsentReason](https://hl7.org/fhir/R4/valueset-data-absent-reason.html)
-- [Encounter Reason Codes](https://hl7.org/fhir/R4/valueset-encounter-reason.html)
-- [EncounterStatus](https://hl7.org/fhir/R4/valueset-encounter-status.html)
-- [EventStatus](https://hl7.org/fhir/R4/valueset-event-status.html)
-- [Immunization Status Codes](https://hl7.org/fhir/R4/valueset-immunization-status.html)
-- [LocationMode](https://hl7.org/fhir/R4/valueset-location-mode.html)
-- [LocationStatus](https://hl7.org/fhir/R4/valueset-location-status.html)
-- [Medication request intent](https://hl7.org/fhir/R4/valueset-medicationrequest-intent.html)
-- [Medicationrequest status](https://hl7.org/fhir/R4/valueset-medicationrequest-status.html)
-- [NameUse](https://hl7.org/fhir/R4/valueset-name-use.html)
-- [Observation Category Codes](https://hl7.org/fhir/R4/valueset-observation-category.html)
-- [Observation Interpretation Codes](https://hl7.org/fhir/R4/valueset-observation-interpretation.html)
-- [Observation Reference Range Meaning Codes](https://hl7.org/fhir/R4/valueset-referencerange-meaning.html)
-- [ObservationStatus](http://hl7.org/fhir/R4/valueset-observation-status.html)
-- [Participant type](https://hl7.org/fhir/R4/valueset-encounter-participant-type.html)
-- [Provenance activity type](https://hl7.org/fhir/R4/valueset-provenance-activity-type.html)
-- [Vital Signs Units](https://hl7.org/fhir/R4/valueset-ucum-vitals-common.html)
-- [VitalSigns](https://hl7.org/fhir/R4/valueset-observation-vitalsignresult.html)
+|ValueSet|Profile where used in|Published|
+|---| ---| ---|
+|[ActEncounterCode - AU Extended](https://build.fhir.org/ig/hl7au/au-fhir-base//ValueSet-au-v3-ActEncounterCode-extended.html)|[AU Core Encounter](StructureDefinition-au-core-encounter.html)|AU Base|
+|[AdministrativeGender](https://hl7.org/fhir/R4/valueset-administrative-gender.html)|[AU Core Patient](StructureDefinition-au-core-patient.html)|FHIR|
+|[Adverse Reaction Agent](https://healthterminologies.gov.au/fhir/ValueSet/adverse-reaction-agent-1)|[AU Core AllergyIntolerance](StructureDefinition-au-core-allergyintolerance.html)|NCTS|
+|[AllergyIntolerance Clinical Status Codes](https://hl7.org/fhir/R4/valueset-allergyintolerance-clinical.html)|[AU Core AllergyIntolerance](StructureDefinition-au-core-allergyintolerance.html)|FHIR|
+|[AllergyIntolerance Verification Status Codes](http://hl7.org/fhir/R4/valueset-allergyintolerance-verification.html)|[AU Core AllergyIntolerance](StructureDefinition-au-core-allergyintolerance.html)|NCTS|
```

Comment:
Change NCTS to AU Base

Reply:
fixed

Reply:
Can't see the change - however I made an error in my review comment. Change from NCTS to FHIR, sorry.

Reply:
fixed. resolve when satisfied with change.